builtin/providers/ns1/permissions.go

```diff
@@ -0,0 +1,184 @@
+package ns1
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"gopkg.in/ns1/ns1-go.v2/rest/model/account"
+)
+
+func addPermsSchema(s map[string]*schema.Schema) map[string]*schema.Schema {
+	s["dns_view_zones"] = &schema.Schema{
+		Type:     schema.TypeBool,
+		Optional: true,
+	}
+	s["dns_manage_zones"] = &schema.Schema{
+		Type:     schema.TypeBool,
+		Optional: true,
+	}
+	s["dns_zones_allow_by_default"] = &schema.Schema{
```

Comment:
Sweet, will do. Thanks again. Had the govendor fix stuff in a different branch(https://github.com/hashicorp/terraform/pull/10391/commits/3dfcfc21265628c22d843f8ea99b51321f50e9cc)... this one just got out of hand